### ISSUE TITLE
Inconsitency in WarpTheory

### ISSUE THREAD

**Issue by TechnicianLP** [GTNewHorizons/NewHorizons#2312]
this really is a issue for https://github.com/Namikon/WarpTheory (which you said you are fixing bugs in in https://github.com/AKA-Syenite/WarpTheory/issues/10 and has its issuetracker disabled)

Thaumcraft v4.2.3.1 states in its changelog: 
"- warp events will now steadily reduce in frequency and eventually stop if you refrain from gaining warp or doing any warp related activities. The moment you gain warp again this resets."

WarpTheory does not adhere to this design choice made by thaumcraft and is therefore inconsitent with handling warp

**Comment by Dream-Master** [comment]
Yes i know. Warp is dangerous and forbidden research have a meaning. If warp effects slowy vanish people will research all forbidden stuff at once and not care warp anymore. With warp theory you need to use soap and bath so long you can make the quest and get one pure tear for purging all the warp. On Asmiiridium Level and when open Kami reserach you are ablt to craft the tears by yourself or the amulet which purge the warp.

**Comment by TechnicianLP** [comment]
It would be good if that information would be added to the thaumonomicon one might expect the normal behaviour - also a reference to the purging of it as well as the info about the quest that gives you a tear as reward would be good

**Comment by Dream-Master** [comment]
@TechnicianLP sure. Can you write a line or two text for me i could it. My english isn't the best.

**Comment by Barhandar** [comment]
There are more sources of warp than just research. Eldritch guardian projectiles, for example. Effective-warp reducing over time doesn't encourage researching everything and forgetting about warp.
